tidb-cloud/recovery-group-get-started.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Get Started with Recovery Groups
+summary: Learn how to create a recovery group in TiDB Cloud and view its details.
+---
+
+# Get Started with Recovery Groups
+
+This document describes how to create a recovery group to protect your databases running on TiDB Cloud Dedicated clusters using the [TiDB Cloud console](https://tidbcloud.com/). It also shows how to view details of a recovery group.
+
+## Prerequisites
+
+A recovery group replicates your databases to another cluster to protect your databases from regional disasters. To create a recovery group, you need to have two TiDB Cloud Dedicated clusters. One cluster hosts the primary databases, and a second cluster hosts the replicas of the primary databases. If you have not done so already, follow the steps in [Create a TiDB Dedicated Cluster](/tidb-cloud/create-tidb-cluster.md) to create the necessary clusters.
+
```

Comment:
In the table of https://docs.pingcap.com/tidbcloud/manage-user-access#project-roles, a row about managing replication groups needs to be added.

Reply:
https://github.com/pingcap/docs/pull/17425/files#diff-aec01ff042ba824dd4674abecfb9edcd6c4704b481b951a3ad287809ae8908d9